### PR TITLE
Make issue numbers in changelog hyperlinks

### PR DESCRIPTION
**Pull request by drewnoakes**
Whenever I read the changelog I want to navigate through to the referenced issues.

### CODE REVIEW

File: GitUI/Resources/ChangeLog.md [hunk 1]
```diff
@@ -2,225 +2,225 @@
 Changelog
 =========
 ### Version 2.49.03 (26 March 2017)
-* Fixed issue #3605. NullReferenceException from CanBeGitUrl when trying to clone git repository from dashboard.
-* Fixed issue #3578. File history and blame not show when path to file contain cyrilic chars bug reproducible.
+* Fixed issue [#3605](//github.com/gitextensions/gitextensions/issues/3605). NullReferenceException from CanBeGitUrl when trying to clone git repository from dashboard.
```
Comment (RussKie) [inline GitUI/Resources/ChangeLog.md]: should it be `[#3605](https://github.com/gitextensions/gitextensions/issues/3605)`?

Comment (drewnoakes) [inline GitUI/Resources/ChangeLog.md]: The omission of the URL scheme means the link adopts the scheme of the page on which it is displayed. Adding `https:` would work too, but it's fine without. You can [test it for yourself](https://github.com/drewnoakes/gitextensions/blob/f2f1fe91397869ad17b4b203cac580308a35fb95/GitUI/Resources/ChangeLog.md).

**Comment by KindDragon** [comment]
Sorry for late response, but I deliberately did not add links to the file because Help>Changelog window doesn't support Markdown and without formatting ChangeLog.md hard to read with links

**Comment by jbialobr** [comment]
Can we point to the changelog on Github instead of showing local file? Alternatively we could convert Changelog.md into html while setup preparation and include the generated html file.

**Comment by vbjay** [comment]
I like the HTML conversion and showing it locally and having a link to the
online version.

**Comment by KindDragon** [comment]
> Alternatively we could convert Changelog.md into html while setup preparation and include the generated html file.

We can do that even in runtime https://github.com/Knagis/CommonMark.NET

**Comment by KindDragon** [comment]
> Can we point to the changelog on Github instead of showing local file?

I just like the idea when GitExt include all necessary for work with Git repositories inside corporative network (without the Internet).

**Comment by vbjay** [comment]
Sure. Why I suggested that we convert it but also have a link that points
to online version too. They can have the option to go online.

**Comment by KindDragon** [comment]
If we do not have time for this now, I think we can temporarily revert these changes

**Comment by jbialobr** [comment]
I reverted this. We should not delay the next release because of this issue.

**Comment by jbialobr** [comment]
I [added](https://github.com/gitextensions/gitextensions/commit/d838293db21c528d277c4cec7c24626d5c1172b7) entries for v2.50 using reference links.
https://github.com/gitextensions/gitextensions/blob/master/GitUI/Resources/ChangeLog.md
What do you think?

**Comment by RussKie** [comment]
Nice!

**Comment by KindDragon** [comment]
This is much better!

**Pull request merged**